test-integration/graql/reasoner/BUILD

```diff
@@ -1,6 +1,6 @@
 java_test(
     name = "reasoning-it",
-    size = "large",
+    size = "enormous",
```

Comment:
maybe we should split the test in the future

Reply:
will do, just trying to find a point where it passes :)

Reply:
👍